### ISSUE TITLE
purpose of this repo?

### ISSUE THREAD

**Issue by matthewberryman** [urbanriskmap/cognicity-sensors-etl-us#1]
Already have https://github.com/urbanriskmap/cognicity-sensors

**Comment by tomasholderness** [comment]
Hi @matthewberryman - we're currently piloting a new approach for sensor data which solves some of the issues we've had in the past around scalability.

cognicity-sensors repo provides a restful API for sensors + sensor data to be stored in database which is platform/sensor agnostic. Each subsequent type of sensor will then require an ETL  module for periodic updates, which will pull data from sensor data source, transform it as required, and push to the new API for storage and access by web products.

**Comment by matthewberryman** [comment]
At the moment cognicity-sensors has an ETL lambda (for v1 of the floodsensors, although v2 will be similar) and in the README.md for cognicity-sensors it says the restful API is to go into cognicity-server per the issue you opened https://github.com/urbanriskmap/cognicity-server/issues/57 
If you want to have cognicity-sensors for a separate (serverless?) API that's fine but in that case I would move the ETL code from cognicity-sensors out, update readme, and I also think this repo should be renamed to cognicity-sensors-etl and capture all the ETLs in different subdirectories.

**Comment by tomasholderness** [comment]
Agreed. I ended up re-purposing the cognicity-sensors repo for the new API, currently on the dev branch. Once it is ready we can move out the flood sensors to its own ETL space - but want to wait until API is stable first.

**Comment by matthewberryman** [comment]
@talltom ah ok, sorry, I hadn't seen the dev branch- my bad. Now it makes sense. Propose renaming this repo and having all the ETLs in one repo? Neither here nor there IMHO.
